jobs/JGLOBAL_MARINE_BMAT

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+
+export DATA=${DATA:-${DATAROOT}/${RUN}marinebmat_${cyc}}
+# source config.base, config.ocnanal and config.marinebmat
+# and pass marinebmat to ${machine}.env
+source "${HOMEgfs}/ush/jjob_header.sh" -e "marinebmat" -c "base ocnanal marinebmat"
+
+# If ensemble perturbations are used, save them in a DATAenspert directory that will
+# exist throughout the DA cycle
+if (( 10#${NMEM_ENS:-0} > 0 )); then
+  export DATAjob="${DATAROOT}/${RUN}marinebmat.${PDY:-}${cyc}"
+  export DATAenspert="${DATAjob}/enspert"
+  if [[ ! -d "${DATAenspert}" ]]; then mkdir -p "${DATAenspert}"; fi
+fi
+
+##############################################
+# Set variables used in the script
+##############################################
+# shellcheck disable=SC2153
+GDATE=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
+export GDATE
+export gPDY=${GDATE:0:8}
+export gcyc=${GDATE:8:2}
+export GDUMP="gdas"
+export GDUMP_ENS="enkf${GDUMP}"
```

Comment:
The need to export these will be interrogated - `gPDY` and `gcyc` in the python scripts derived from `self.task_config.previous_cycle`